### ISSUE TITLE
Getting the gap-minder data and order of lessons

### ISSUE THREAD

**Issue by hdashnow** [COMBINE-Australia/r-novice-gapminder#1]
The gapminder data is first used in 04-data-structures
But it needs to be downloaded, and loaded as a data frame before that. 
I could put this in  04-data-structures, but it might make more sense to download the data when giving them a tour of R studio, e.g. as part of setting up an R project. 

Note, the 06-reading-data could be moved earlier to just before 04-data-structures to help with this.
I haven't yet come across the bit where we download the data, but I'm probably not looking hard enough!

**Comment by hdashnow** [comment]
Oh, I found a comment on downloading the data here:
http://combine-australia.github.io/r-novice-gapminder/instructors.html

I think we can do better, though.

**Comment by griffinp** [comment]
I agree Harriet and I seem to remember that method of accessing the data caused some problems last time. Is there a way we can host it somewhere without having to get people to access it via git?

**Comment by griffinp** [comment]
Also I moved the reading-data section to before the data-structures as suggested

**Comment by hdashnow** [comment]
We could use Git (they already have it installed from today).
An alternative would be to host the data only in a separate repository, then they can download it as a zip file if they run into any Git troubles.

**Comment by hdashnow** [comment]
Okay, so here's the plan:
Noel will download the data and put it in a "data" directory when he's setting up R-studio.
I will split up the data type lessons into two parts. The first part doesn't mention gapminder. The second part starts by loading gapminder into memory (it's already in the data directory) and then continue from there.

**Comment by hdashnow** [comment]
I've separated out the data structures lessons into two parts. Noel will do 04, I'll do 05.

**Comment by hdashnow** [comment]
Okay, just fixed index and pushed. So everything is set for Noel to keep editing.

Still need to add a bit about where the data is.
